### PR TITLE
Migrate Microsoft Azure configuration page to Bootstrap

### PR DESCRIPTION
**Pull request by eduardoj**
Before:

Screenshot from 2019-06-03 16-44-04_azure_before_1

Screenshot from 2019-06-03 16-44-46_azure_before_2

After:

Screenshot from 2019-06-03 16-45-39_azure_after_1

Screenshot from 2019-06-03 16-46-16_azure_after_2

### CODE REVIEW

**Comment by dmarcoux** [comment]
Link to review: https://obs-reviewlab.opensuse.org/eduardoj-bootstrap_cloud_upload_ec2/cloud/azure/configuration

File: src/api/app/controllers/webui/cloud/azure/configurations_controller.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ class ConfigurationsController < WebuiController
 
         # GET /cloud/azure/configuration
         def show
+          # FIXME: remove it when cloud upload are ready to production
+          switch_to_webui2 if Rails.env.development?
```
Comment (dmarcoux) [inline src/api/app/controllers/webui/cloud/azure/configurations_controller.rb]: For this action, the view is already fully migrated in this PR. We don't need to check for the environment.

**Comment by eduardoj** [comment]
[Cloud Upload Azure](https://trello.com/c/EmfT9Dg6/1157-cloud-upload-azure)

File: src/api/app/controllers/webui/cloud/azure/configurations_controller.rb (hunk 1 above)
Comment (eduardoj) [reply src/api/app/controllers/webui/cloud/azure/configurations_controller.rb]: Check removed. Comment removed too.

File: src/api/app/views/webui2/webui/cloud/azure/configurations/_breadcrumb_items.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+= render partial: 'webui/main/breadcrumb_items'
+
+%li.breadcrumb-item
+  = link_to 'Cloud Upload', cloud_upload_index_path
+%li.breadcrumb-item
+  = link_to 'Cloud Configuration', cloud_configuration_index_path
+%li.breadcrumb-item.active
```
Comment (dmarcoux) [inline src/api/app/views/webui2/webui/cloud/azure/configurations/_breadcrumb_items.html.haml]: I just realized, we're missing `{ 'aria-current' => 'page' }` for accessibility.

Comment (eduardoj) [reply src/api/app/views/webui2/webui/cloud/azure/configurations/_breadcrumb_items.html.haml]: `{ 'aria-current' => 'page' }` added.

**Pull request merged**